.github/workflows/build.yml

```diff
@@ -24,37 +23,99 @@ jobs:
                     - '8.1'
                 symfony-version:
                     - '4.4.*'
-                    - '5.3.*'
                     - '5.4.*'
                     - '6.0.*'
                 exclude:
                     - php-version: '7.4'
                       symfony-version: '6.0.*'
         steps:
             - name: Checkout
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
 
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
                   coverage: none
-                  ini-values: "memory_limit=-1"
+                  ini-values: "memory_limit=-1, zend.assertions=1"
                   php-version: ${{ matrix.php-version }}
                   tools: composer:v2, flex
 
+            # This works around SYMFONY_REQUIRE currently not working (https://github.com/symfony/flex/issues/946):
+            - name: Lock Symfony version
+              run: VERSION=${{ matrix.symfony-version }} .github/workflows/lock-symfony-version.sh
+
+            - name: Install dependencies
+              run: |
+                  composer config --no-plugins allow-plugins.dealerdirect/phpcodesniffer-composer-installer true
+                  composer install --prefer-dist --no-progress --no-plugins
+              #env:
+              #    SYMFONY_REQUIRE: "${{ matrix.symfony-version }}"
+
+            - name: Run tests
+              run: composer test
+
+    psalm:
+        name: Run Psalm
```

Comment:
I don't think we need to run this in all matrix versions 🌱 . When PHP 7.4 and its available dependencies are fine, the other versions should be as well.